incubator/hnc/config/default/manager_auth_proxy_patch.yaml

```diff
@@ -26,3 +26,4 @@ spec:
         - "--leader-election-id=hnc-controller-leader-election-helper"
         - "--max-reconciles=10"
         - "--apiserver-qps-throttle=50"
+        - "--enable-internal-cert-management=true"
```

Comment:
Do we need this line in any other yaml file? Eg I think this list of arguments is included in another file too.

Reply:
I don't think so. I just double checked the PRs [1](https://github.com/kubernetes-sigs/multi-tenancy/commit/29c17f2757803f0e28e07e99ceaba0f1c59b57e2), [2](https://github.com/kubernetes-sigs/multi-tenancy/pull/377/commits/39010d8bb51978fc392e5e99b2c56e86292d5c14) for `max-reconciles` and `apiserver-qps-throttle` to confirm that they are only used here.

Reply:
sgtm, thanks for checking.